web/src/ee/features/evals/server/router.ts

```diff
@@ -545,7 +556,7 @@ export const evalRouter = createTRPCRouter({
       z.object({
```

Comment:
Note: Once you merge main you'll see that we also call `updateEvalJob` [here](https://github.com/langfuse/langfuse/blob/882ad83aad92816d5c630571be58d0823b610056/web/src/ee/features/experiments/components/CreateExperimentsForm.tsx#L281), we will need to reformat the input slightly there too :)

Reply:
Done, thanks!